WebDriverAgentLib/Utilities/FBXCodeCompatibility.m

```diff
@@ -106,7 +107,8 @@ - (XCUIElement *)fb_firstMatch
     XCUIElement* result = self.firstMatch;
     return result.exists ? result : nil;
   }
-  if (!self.element.exists) {
+
+  if (![self.element waitForExistenceWithTimeout:FB_FIRST_MATCH_WAIT_EXITS_INTERVAL]) {
```

Comment:
this would be a breaking change. Please add the timer directly to the functional test where it is necessary

Reply:
yeah, true. misunderstood your suggestion